### DISCUSSION TITLE
Filtering on schema objects

### DISCUSSION THREAD

**Discussion by tomschut** [maoosi/prisma-appsync#114] [Ideas]
The generator currently generates a full set of resolvers and mappings for every object in the schema. It'd be nice if there were some kind of filter to exclude e.g. Mutations, or specific resources.

**Comment by maoosi** [comment]
There is already something built-in to customise the GraphQL schema output (to some extent).

I am currently writing documentation for this and will add it as part of the next release.

In the meantime, you can find examples here: https://github.com/maoosi/prisma-appsync/issues/88#issuecomment-1372066326

**Comment by maoosi** [comment]
The updated documentation will be release as part of [1.0.0-rc.6](https://github.com/maoosi/prisma-appsync/milestone/5), but it can already be found [here](https://github.com/maoosi/prisma-appsync/blob/dev/docs/features/gql-schema.md).
